package.json

```diff
@@ -73,5 +74,8 @@
         }
       ]
     ]
+  },
+  "browser": {
```

Comment:
I think this supposed to replace xhr with request module but I'm not seeing how this used or why.

Reply:
Oops - I was originally using that package for ajax requests, but removed it.